.github/workflows/gradle.yml

```diff
@@ -40,3 +40,9 @@ jobs:
           slug: embrace-io/opentelemetry-kotlin
           fail_ci_if_error: false
           files: build/reports/kover/reportRelease.xml
+
+      - name: Build Example Apps
+        run: ./gradlew :examples:jvm-app-java-api:build :examples:jvm-app-kotlin-api:build :examples:android-app-java-api:assembleRelease :examples:android-app-kotlin-api:assembleRelease --stacktrace
+
+      - name: Build Telescope App
```

Comment:
Can this be altered so that it is included in the main project and can build that way?